main/requirements.txt

```diff
@@ -0,0 +1,15 @@
+blinker==1.4
+flask-login==0.4.1
+flask-oauthlib==0.9.5
+flask-restful==0.3.6
+flask-wtf==0.14.2
+flask==1.0.2
+jinja2==2.10
+pyjwt==1.6.4
+unidecode==1.0.22
+webargs==4.1.2
+setuptools==40.5.0
+google-cloud==0.34.0
+google-cloud-vision==0.34.0
+#google-appengine==1.5.1
```

Comment:
hmmm.. do we need it here?

Reply:
Yes. In python 3 environment you tell it what libs to use when building a container via main/requirements.txt and it uses them.

Reply:
but why it's commented out?

Reply:
The point is that `google-appengine` *will not* be available in Python 3.7 standard environment, so migrating will involve changing all uses of this library to something else…

Reply:
Yes, we will have to change all the datastore code to use `google.cloud` instead of `google.appengine`.